css/main.css

```
@@ -202,12 +212,305 @@ footer {
   gap: 1.2rem;
 }
 
+/* PC: 1200px 이상 */
+@media (min-width: 1200px) {
+  .header__inner {
+    padding: 0.9rem 20rem;
+  }
+
+  .section__card--img {
+    width: 60%;
+    position: relative;
+  }
+
+  .section__card--img > img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+}
+
+/* 대형 PC: 1920px 이상 */
 @media screen and (min-width: 1920px) {
   .header__inner {
     padding: 0 40rem;
   }
 
+  .main__background {
+    margin: 0 40rem;
+  }
+
+  section:nth-of-type(2),
+  section:nth-of-type(3),
+  section:nth-of-type(4) {
+    padding: 13.8rem 40rem;
+  }
+
+  .main__background--footer > div {
+    margin: 0 40rem;
+  }
+
+  .section__card {
+    gap: 6.4rem;
+  }
+
   footer {
     padding: 3.2rem 40rem;
   }
 }
+
+/* Tablet: 768px ~ 1199px 이하 */
+@media (min-width: 768px) and (max-width: 1199px) {
+  .header__inner {
+    padding: 1rem 2.4rem;
+  }
+
+  .main__background {
+    margin: 0;
+    flex-direction: column;
+    width: 100%;
+    height: auto;
+    align-items: center;
+  }
+
+  .main__background--text h2 > br {
+    display: none;
+  }
+
+  .main__background--text {
+    margin: 0 auto;
+    min-width: 51.2rem;
+    padding-top: 8.4rem;
+  }
+
+  .btn__large {
+    padding: 1.6rem 12.4rem;
+  }
+
+  .main__background--text .btn__large {
+    width: 35.7rem;
+    margin: 2.4rem auto 21.1rem;
+  }
+
+  .section__card {
+    justify-content: center;
+    height: auto;
+    flex-direction: column;
+    gap: 2.4rem;
+  }
+
+  section:nth-of-type(2) {
+    padding: 2.4rem 2.4rem 0;
+  }
+
+  section:nth-of-type(3) {
+    margin: 5.2rem 0;
+    padding: 0 2.4rem 0;
+  }
+
+  section:nth-of-type(4) {
+    padding: 0 2.4rem 0;
+  }
+
+  section:nth-of-type(2) .section__card,
+  section:nth-of-type(4) .section__card {
+    align-items: flex-start;
+    width: 100%;
+  }
+
+  section:nth-of-type(3) .section__card {
+    flex-direction: column-reverse;
+    align-items: flex-end;
+  }
```

P2:
디자인대로 화면에 꽉 차게 나오려면 위에서 설정해주신 width 값을 덮어씌워야합니다.

```suggestion
  section:nth-of-type(3) .section__card {
    flex-direction: column-reverse;    
    align-items: flex-end;
    width: 100%;
  }
```